x-pack/plugins/observability_onboarding/public/components/app/home/logs_onboarding_wizard/icons/opentelemetry_svg.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export function OpentelemetrySvg() {
```

Comment:
Is there a benefit of defining these as components instead of .svg files?

Reply:
These are probably not needed for the first iteration anyways, will remove